### ISSUE TITLE
UI for Anchor Token, Pool Tokens, and Interest Rate

### ISSUE THREAD

**Issue by anastasiabusygina** [chainhackers/ethonline-2023#8]
![Image](https://github.com/chainhackers/ethonline-2023/assets/89840876/f375d81c-f65b-4eb7-b8d7-7675ad1c0c19)

![Image](https://github.com/chainhackers/ethonline-2023/assets/89840876/962cc57a-a7f5-4a21-ac5b-22f391436842)

### Objective
Develop an interface for the user to:

- Select an anchor token.
- Add pool tokens (with the anchor token already selected and included).
- The native network token is also included in the pool by default. Therefore, if the anchor token is not equivalent to the native network token, the pool will immediately contain two tokens: the native network token and the anchor currency.
- Input an interest rate.
- Have the ability to navigate to a preview of the selected parameters in a modal window (details of the modal are covered in a separate task)
- Log user interactions in the browser's console.

#### Dropdown for anchor token selection

- [x] Contains options with various anchor tokens.
- [x] Only one anchor token can be selected at a time.
- [x] Upon selection, log: `Anchor token selected: [token name]` in the console.

#### Button to add pool tokens

- [x] If no anchor token is selected, this feature is disabled.
- [x] The selected anchor token is automatically included in the pool tokens set.
- [x] The native network token is automatically included in the pool tokens set.
- [x] The button becomes active only after the anchor token is selected.
- [x] The button to add tokens is disabled if 20 tokens have already been added.
- [x] Tokens can be removed from the list, except for the anchor token and the native network token.
- [x] On button click, log: `Pool tokens added: [token name]`.

#### Input field for interest rate

- [x] Accepts only numeric values. (limitations?)
- [x] Input logs: `Interest rate input: [rate value]` in the console.

#### Button to display selected parameters' preview in a modal

- [x] Becomes active when the anchor token is selected, there are at least two tokens in the pool, and the "interest rate" field is correctly filled out.
- [ ] Click logs: `Request to display parameter preview`.
- [ ] The preview modal details are covered in a separate task.

#### Logging actions

- [x] All user interactions with the interface should be logged in the browser's console.

### Additional Notes

- [x] The interface is for web only; responsiveness is not required.
- [x] Implement input data validation.
